### DISCUSSION TITLE
FeatherWing - 3.5" 480x320 Touchscreen for Feathers - V2

### DISCUSSION THREAD

**Discussion by My-Random-Thoughts** [HASwitchPlate/openHASP#588] [Hardware]
Looking at https://www.adafruit.com/product/3651, there is a change:

> As of Nov 15, 2023 we've made a big redesign to this product: We now use the TSC2007 instead of the discontinued STMPE811 touchscreen controller. The screen and micro SD card are the same but any touchscreen code will need to be updated to use our Arduino or CircuitPython library. We've also updated the reset button to be right-angle and added a STEMMA QT port.

Any idea on how long it will take to support this new configuration?  Just want to ask before I buy it  :)
Thank you.

**Comment by amanaplan** [comment]
I've added support for the 2.4" 320x240 TFT Featherwing v2 with the TSC2007 touch controller in my fork (https://github.com/amanaplan/openHASP).  Adding support for the 3.5" version would be very simple.

**Reply by fvanroie** [reply]
Please introduce a Pull Resuest so this board can be integrated in the master branch. Thanks.

**Reply by amanaplan** [reply]
Added PR https://github.com/HASwitchPlate/openHASP/pull/613 (Add support for updated Adafruit 2.4 and 3.5 TFT FeatherWings and HUZZAH32 board). :)

**Comment by amanaplan** [comment]
@My-Random-Thoughts Support for these featherwings has been merged into [master](https://github.com/HASwitchPlate/openHASP/tree/master).

**Comment by My-Random-Thoughts** [comment]
Thank you all.
